apps/api/src/graphql/resolvers/notification.ts

```
@@ -0,0 +1,69 @@
+});
+
+const CommentNotificationData = builder.objectRef<CommentNotificationData>('CommentNotificationData').implement({
+  fields: (t) => ({
+    comment: t.field({
+      type: Comment,
+      resolve: (data) => data.commentId,
+    }),
+  }),
+});
+
+const AnnouncementNotificationData = builder.objectRef<AnnouncementNotificationData>('AnnouncementNotificationData').implement({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'AnnouncementNotificationData'. Consider to delete it or rename it.

'AnnouncementNotificationData' is defined here:


(lint/suspicious/noRedeclare)